pkg/shell/manifests.ts

```diff
@@ -19,18 +19,146 @@
 
 import { JsonValue, JsonObject } from "cockpit";
 
+// Generic validation machinery
```

Comment:
This feels like pkg/lib/, at least eventually? Of course fine to start introducing it into the shell. I just find the `import_*` a bit strange, perhaps `json_get_*` or `get_json_*`?

Reply:
> This feels like pkg/lib/, at least eventually?

Yes.